### DISCUSSION TITLE
Hi Mr.Daniel , I'm very much intrested in learning of PyTorch once i started coding with you and excited to learn the thing about still what the PyTorch does with you , I'm stuck with the error please help me out with this error  #LoveFromINDIA

### DISCUSSION THREAD

**Discussion by arbazz-shaik** [mrdbourke/pytorch-deep-learning#750] [Q&A]
<img width="820" alt="Screenshot 2023-12-08 161836" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/147133765/49fd54fe-1686-4dbc-ae95-a7115cdbbf2a">

**Comment by arbazz-shaik** [comment]
# Turn data into tensors
# Otherwise this causes issues with computations later on
import torch
X_np = np.array(X)
y_np = np.array(y)

X = torch.from_numpy(X_np).type(torch.float)
y = torch.from_numpy(y_np).type(torch.float)

# View the first five samples
X[:5], y[:5]

Sir i got the output , Thanks for your Knowledge sharing sir , which got me more intrest in learning about the PyTorch and the way of your #Teaching LOVEIT 💯 ;)

**Comment by shisirkha** [comment]
The problem you are facing is that you run the cell 1 time then if you do run the cell again it will cause an error that `expected <nd.array> but found <torch.Tensor> type`.

Because when you run the cell ones it saves the value as a tensor then when you run the cell again it sees that it is no longer `nd.array` instead it is a `torch.Tensor`

Good luck solving this
